notebooks/00-getting_started.ipynb

```diff
@@ -107,7 +109,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "model_name = 'default_SEM_model'"
+    "model_name = 'model_seg_rat_axon-myelin_sem'"
```

Comment:
At line 103 above, the names of the model should be updated.

Reply:
Resolved in 2dbc5f768